client/core/trade.go

```diff
@@ -503,7 +584,7 @@ func (t *trackedTrade) tick() (int, error) {
 		// notifications before swapMatches.
 		corder, _ := t.coreOrderInternal()
 		if err != nil {
-			errs.add("swapMatches: %v", err)
+			errs.addErr(err)
```

Comment:
The `err` returned from `swapMatches()` already contain the `"swapMatches - ..."` prefix.